### ISSUE TITLE
Some ADC yellow blocks have non-intuitive interfaces and/or no/limited simulation behaviour

### ISSUE THREAD

**Issue by amartens** [casper-astro/mlib_devel#166]
Some ADC yellow blocks contain no support for simulation during model creation i.e there is no way to input simulation and/or control data, the only ports are output.
The second part of this is that some have ports that are non-intuitive to use, and don't match up with other CASPER blocks.

**Comment by AdamI75** [comment]
@amartens Yes, this is to be expected. Tutorial 4 Wideband Spectrometer - non DDC bypass will not work. This needs to either be removed or the correct Githash needs to be added that has the old yellow block. The ADC yellow block for SKARAB now has a master and slave parameter for synchronisation. I think @mchirindo should update this tutorial with the new SKARAB yellow block. Tutorial 5 also links to our repo and not the casper-astro. This must be fixed.
